README.md

```diff
@@ -135,18 +139,22 @@ from `soss_profile_client` to `soss_profile_server`.
 
 ### YAML dynamic types
 
-*eProsima*'s *dtparser* can create dynamic types from a YAML map,
-allowing the user to define new dynamic types for each run without the need of rebuilding the project.
+*eProsima*'s *xtypes::idl::Parser* can create xtypes from an IDL defined in the YAML file,
```

Comment:
Maybe is no longer necessary go deeply in this direction because now the who parser is soss-core, not soss-dds

Reply:
I think so too.

Reply:
Could you remove these two lines? I suggets them
```suggestion
```